src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -226,6 +228,35 @@ public OptionData setRequired(boolean required)
         return this;
     }
 
+    /**
+     * Add a predefined choice for this option.
+     * <br>The user can only provide one of the choices and cannot specify any other value.
+     * 
+     * @param  name
+     *         The name used in the client
+     * @param  value
+     *         The value received in {@link net.dv8tion.jda.api.interactions.commands.OptionMapping OptionMapping}
+     * 
+     * @throws IllegalArgumentException
+     *         If the name is null, empty or longer than 100 Characters.
+     *         Also thrown if this is not an option of type {@link OptionType#NUMBER} or more than 25 choices are provided.
+     * 
+     * @return The OptionData instance, for chaining
+     */
+    @Nonnull
+    public OptionData addChoice(@Nonnull String name, double value)
+    {
+        Checks.notEmpty(name, "Name");
+        Checks.notLonger(name, 100, "Name");
+        Checks.check(value >= -0x1.0p53, "Double value may not be lower than -2^53");
+        Checks.check(value <= 0x1.0p53, "Double value may not be larger than 2^53");
```

Comment:
These should be public constants with docs references

Reply:
Would it in that case not also be a good idea to use a constant for the name?